lints.toml

```diff
@@ -27,37 +25,31 @@ deny = [
     'clippy::dbg_macro',
     'clippy::else_if_without_else',
     'clippy::enum_glob_use',
-    # This is 99% not needed
-    # 'clippy::inline_always',
+    'clippy::inline_always',
     'clippy::let_underscore_drop',
     'clippy::let_unit_value',
     'clippy::match_on_vec_items',
     'clippy::match_wild_err_arm',
+    # In crypto code, it is fairly common to have similar names e.g. `owner_pk` and `owner_k`
     # 'clippy::similar_names',
     'clippy::needless_borrow',
 
     # style
+    'clippy::style',
     'clippy::explicit_into_iter_loop',
     'clippy::explicit_iter_loop',
     'clippy::if_not_else',
     'clippy::match_bool',
-    # perhaps this is a bit harsh
-    # 'clippy::too_many_lines',
+    'clippy::needless_pass_by_value',
+    'clippy::range_plus_one',
+    'clippy::struct_excessive_bools',
+    'clippy::too_many_lines',
     'clippy::trivially_copy_pass_by_ref',
 
     # casting mistakes
     'clippy::cast_lossless',
-    # 'clippy::cast_possible_truncation',
-    # 'clippy::cast_possible_wrap',
-    # 'clippy::cast_precision_loss',
-    # This is tricky because sqlite uses signed ints and it's used to represent unsigned data
-    # 'clippy::cast_sign_loss'
+    'clippy::cast_possible_truncation',
+    'clippy::cast_possible_wrap',
+    'clippy::cast_precision-loss',
+    'clippy::cast_sign_loss'
 ]
-
-allow= [
-    # allow Default::default calls
-    'clippy::default_trait_access',
- # Generally when developers fix this, it can lead to leaky abstractions or worse, so
- # too many arguments is generally the lesser of two evils
- 'clippy::too_many_arguments'
```

Comment:
I didn't realise this was removed from `meta`. It should be there as well

Reply:
Good catch, will update here and in meta